committee-steering/governance/sig-governance.md

```diff
@@ -69,6 +69,12 @@ Subproject Owner Role.  (this different from a SIG or Organization Member).
   - Number: 2-3
   - Membership tracked in [sigs.yaml]
 
+- Subproject Leads
```

Comment:
I think "lead" would be extremely confusing. Project manager?

I also think it has to be a suggested role, for larger, more active subprojects. kube-proxy, for instance, probably wouldn't need one.

It was also suggested that points of contact would be useful.

Reply:
What I was looking for in the conversation last week was simple lines of communication for a sub-project to bootstrap new folks.  A slack channel is sufficient for that purpose and is can add a slack channel for each subproject then I think the current problems are solved.